snmp/datadog_checks/snmp/data/default_profiles/aruba-cx-switch.yaml

```diff
@@ -0,0 +1,127 @@
+extends:
+  - aruba.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.47196.4.1.1.1.*
+  - 1.3.6.1.4.1.47196.4.1.1.1.1
+  - 1.3.6.1.4.1.47196.4.1.1.1.2
+  - 1.3.6.1.4.1.47196.4.1.1.1.3
+  - 1.3.6.1.4.1.47196.4.1.1.1.5
+  - 1.3.6.1.4.1.47196.4.1.1.1.50
+  - 1.3.6.1.4.1.47196.4.1.1.1.70
+  - 1.3.6.1.4.1.47196.4.1.1.1.101
+  - 1.3.6.1.4.1.47196.4.1.1.1.103
+  - 1.3.6.1.4.1.47196.4.1.1.1.104
+  - 1.3.6.1.4.1.47196.4.1.1.1.150
+  - 1.3.6.1.4.1.47196.4.1.1.1.304
+  - 1.3.6.1.4.1.47196.4.1.1.1.405
+metrics:
+  - MIB: HOST-RESOURCES-MIB
+    symbol:
+      name: memory.total
+      OID: 1.3.6.1.2.1.25.2.3.1.5.1
```

Comment:
Do you know why we specifically choose index 1 of 1.3.6.1.2.1.25.2.3.1.5 OID?

Reply:
not sure why the choice of index 1